app/src/main/java/com/example/gradeassist/MainActivity.kt

```diff
@@ -25,5 +26,10 @@ class MainActivity : AppCompatActivity() {
         )
         setupActionBarWithNavController(navController, appBarConfiguration)
         navView.setupWithNavController(navController)
+        val fragment = HomeFragment()
+        val transaction = supportFragmentManager.beginTransaction()
+        transaction.replace(R.id.container, fragment)
+        transaction.commit()
```

Comment:
to enable fragment switching

Reply:
Is this required at this time?  If not, it's extra work... or unneeded code.  
I don't see anything that requires a fragment swap here, so it should use the default fragment.  I would not include this code until/unless a fragment swap is actually required.